packages/@n8n/nodes-langchain/nodes/trigger/ChatTrigger/ChatTrigger.node.ts

```diff
@@ -241,6 +240,17 @@ export class ChatTrigger extends Node {
 				placeholder: 'Add Field',
 				default: {},
 				options: [
+					// CORS parameters are only valid for when chat is used in hosted or webhook mode
+					...(Array.isArray(commonCORSParameters)
+						? commonCORSParameters.map((p: INodeProperties) => ({
+								...p,
+								displayOptions: {
+									show: {
+										'/mode': ['hostedChat', 'webhook'],
+									},
+								},
+							}))
+						: []),
```

Comment:
As `commonCORSParameters` is const array - do we really need to check it?
```suggestion
					...commonCORSParameters.map((p: INodeProperties) => ({
						...p,
						displayOptions: {
							show: {
								'/mode': ['hostedChat', 'webhook'],
							},
						},
					})),
```